src/solver/optimisation/opt_alloc_probleme_a_optimiser.cpp

```diff
@@ -176,8 +183,10 @@ static void optimisationAllocateProblem(PROBLEME_HEBDO* ProblemeHebdo, const int
       = (double*)MemAlloc(ProblemeAResoudre->NombreDeVariables * sizeof(double));
     ProblemeAResoudre->Colonne = (int*)MemAlloc(ProblemeAResoudre->NombreDeVariables * sizeof(int));
 
-    ProblemeAResoudre->NomDesVariables.resize(ProblemeAResoudre->NombreDeVariables);
-    ProblemeAResoudre->NomDesContraintes.resize(ProblemeAResoudre->NombreDeContraintes);
+    ProblemeAResoudre->NomDesVariables
```

Comment:
Because C++ object aren't destroyed (as in `~Object()`) when the memory is managed through `malloc/free` as it is here. It would be if the containing class was managed through `new/delete`.